### PR TITLE
Add option to not use Powerline glyphs

### PR DESCRIPTION
**Pull request by caesar**
Adds the option requested in #38.

Seems to be quite simple, though it's possible I've missed some.

I hope this will be accepted as it would be great for compatibility at very little cost.

Let me know if you'd like me to do anything differently.

### CODE REVIEW

File: fish_prompt.fish [hunk 1]
```diff
@@ -619,13 +620,23 @@ function fish_prompt -d 'bobthefish, a fish theme optimized for awesome'
...
-  set -l __bobthefish_left_arrow_glyph        \uE0B3
+  set -l __bobthefish_branch_glyph            ''
+  set -l __bobthefish_ln_glyph                ''
+  set -l __bobthefish_padlock_glyph           ''
+  set -l __bobthefish_right_black_arrow_glyph ''
+  set -l __bobthefish_right_arrow_glyph       ''
+  set -l __bobthefish_left_black_arrow_glyph  ''
+  set -l __bobthefish_left_arrow_glyph        ''
+
+  if [ "$theme_powerline_fonts" != "no" ]
```
Comment (derekstavis) [inline fish_prompt.fish]: I would prefer making powerline fonts the default, as this would have less performance penalties on the prompt at every render.

Comment (caesar) [inline fish_prompt.fish]: @derekstavis Sure, makes sense. Done.

Comment (bobthecow) [inline fish_prompt.fish]: 👍

File: fish_prompt.fish [hunk 2]
```diff
@@ -627,6 +628,16 @@ function fish_prompt -d 'bobthefish, a fish theme optimized for awesome'
   set -l __bobthefish_left_black_arrow_glyph  \uE0B2
   set -l __bobthefish_left_arrow_glyph        \uE0B3
 
+  if [ "$theme_powerline_fonts" = "no" ]
+    set __bobthefish_branch_glyph            ''
```
Comment (bobthecow) [inline fish_prompt.fish]: we don't use most of the non-arrow glyphs, but we do use branch. is there a good unicode character  that approximates this?

**Comment by bobthecow** [comment]
What should happen if someone disables powerline fonts but enables nerd fonts?

File: fish_prompt.fish (hunk 2 above)
Comment (caesar) [inline fish_prompt.fish]: What about ⎇ (U+2387, ALTERNATIVE KEY SYMBOL)?
However it's not present in all fonts, I believe.
(As a matter of fact I don't think it's important to show the branch icon in 'fallback' mode. It is fairly clear what the branch name means anyway.)

**Comment by caesar** [comment]
Interesting. Why would they do that? :-)
I think nerd fonts include powerline anyway don't they?

**Comment by bobthecow** [comment]
I'm not saying it's sane, just that it's possible given the set of theme configuration variables :)

File: fish_prompt.fish (hunk 2 above)
Comment (bobthecow) [inline fish_prompt.fish]: Let's start with ⎇ and refine later if necessary?

**Comment by caesar** [comment]
Added ⎇ symbol.

As for disabling powerline but enabling nerd fonts, though it would be an odd thing to do, I think the current behaviour matches expectations. It enables nerd fonts (other than the powerline glyphs), but disables powerline.
The alternative would be for the nerd option to override the powerline one, but I don't think that would match expectations.
That said, this is all a hypothetical edge case anyway because there seems to be no valid reason to both enable nerd and disable powerline.

File: fish_prompt.fish [hunk 3]
```diff
@@ -627,6 +628,16 @@ function fish_prompt -d 'bobthefish, a fish theme optimized for awesome'
   set -l __bobthefish_left_black_arrow_glyph  \uE0B2
   set -l __bobthefish_left_arrow_glyph        \uE0B3
 
+  if [ "$theme_powerline_fonts" = "no" ]
+    set __bobthefish_branch_glyph            '\u2387'
```
Comment (bobthecow) [inline fish_prompt.fish]: let's do this without the quotes to match the rest of 'em.

otherwise, :shipit: !

**Comment by bobthecow** [comment]
Oh, please add the config option to the bottom of README as well.

**Comment by caesar** [comment]
👍

**Pull request merged**